sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -64,6 +66,9 @@
     private final AnalysisMetaData analysisMetaData;
     private static final EnumSet<Join.Type> ALLOWED_JOIN_TYPES =
         EnumSet.of(Join.Type.CROSS, Join.Type.INNER, Join.Type.LEFT, Join.Type.RIGHT, Join.Type.FULL);
+    private static final Relation SYS_CLUSTER_SOURCE = new Table(new QualifiedName(
```

Comment:
Since the only usage requires `List<Relation>` I think the constant here could be changed to a List?